### ISSUE TITLE
F - Implement language setting in front end (DevOps #4624)

### ISSUE THREAD

**Issue by takashi-uk** [takashi-uk/test#152]
**Original WorkItem:** DevOps #4624
**State:** Closed

**Comment by takashi-uk** [comment]
**James Brightman** wrote:

<div>Partially blocked now by lack of settings gql query/mutation accessibility.</div>
